engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -104,15 +110,16 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
 
   when (NotStarted) {
     case Event(RunWorkflow, data) =>
-      log.info(s"$Tag: Launching workflow")
-      val submitMessage = SubmitWorkflowCommand(source)
-      workflowManager ! submitMessage
+      log.info(s"$Tag: Submitting workflow")
+      workflowStore ! SubmitWorkflow(source)
```

Comment:
I would think for the SingleWorkflowRunner case, you're actually fine to continue submitting directly to the WMA... 

Unless you want single runner to restart existing workflows, which IMO it shouldn't